### DISCUSSION TITLE
Spellchecking

### DISCUSSION THREAD

**Discussion by GeoRW** [GeoRW/ACBF-Editor#18] [Ideas]
One of the things I'm missing in Editor is spellchecking for translated text. I've just added texts export into GTK3 version, so I can open generated text file in some word processing program and run it through spellchecker. We can research how easy/tough it would be to add spellchecking. At least texts exports should be added to GTK4 version too.

**Comment by mizaki** [comment]
A quick search and it looks like: https://pypi.org/project/pygtkspellcheck/ is promising.

The export kind of goes with being able to import (BubbleTranslator) and specifying a plain text format for that makes sense. It could be as easy as a newline = next text bubble but probably better to use some more solid delimiter. Also possible to use something like JSON, TOML, Markdown etc.

Adding copy and paste buttons too.

A simple text version:
```
1. My text for first text bubble
2. Text for second
```

**Reply by GeoRW** [reply]
That library looks cool, I won't be adding it to GTK3 version, but you can go ahead implementing spellchecking to GTK4.
Regarding export I've added page just page number there, will add also bubble number. In GTK4 you can specify something better export/import compatible. And depends what Bubble Translator can support.
